src/components/Posts/PostDetails/PostDetails.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { AddCommentForm } from '../../Comments/addCommentForm';
+import './PostDetails.scss';
+import { getPostDetails } from '../../../api/posts';
+import {
+  getPostComments,
+  addComment,
+  removeComment,
+} from '../../../api/comments';
+import { CommentsDetails } from '../../Comments/CommentsDetails';
+
+export const PostDetails = ({ postId }) => {
+  const [comments, setComments] = useState([]);
+  const [post, setPost] = useState(null);
+
+  const add = async(newComment) => {
+    await addComment(newComment);
+
+    setComments([...comments, newComment]);
+  };
+
+  const remove = async(removedCommentId) => {
+    await removeComment(removedCommentId);
+    const filteredComments = comments.filter(
+      comment => removedCommentId !== comment.id,
+    );
+
+    setComments(filteredComments);
```

Comment:
add or remove the comment and then fetch from the server the updated version of comments, no need to update the state locally